### DISCUSSION TITLE
Chunk offset?

### DISCUSSION THREAD

**Discussion by thorlucas** [joshuajbouw/bevy_chunk_tiles#147] [General]
I feel that it's a little bit strange that chunks are offset from the center. I feel that chunk `(0, 0, 0)` with a chunk size of `(16, 16, 16)` should be responsible for tiles from `(0, 0, 0)` to `(15, 15, 15)` inclusive, no? Otherwise we have to do a little bit of extra math for the conversion and requires extra mental gymnastics. Additionally, using this scheme would allow us to do some shorthands. For chunk sizes of 16, we could more easily do something like `chunk_point.x << 4 + offset.x`, which feels more natural to me (and is certainly faster) than something like `chunk_point.x * CHUNK_WIDTH - (CHUNK_WIDTH / 2) + offset.x`. Thoughts?

**Comment by joshuajbouw** [comment]
Not a bad idea. Makes sense. If you want to start a PR that would be awesome :D.

**Reply by thorlucas** [reply]
I can give it a shot!
